templates/slider.jsx

```diff
@@ -177,6 +177,7 @@ export default function Slider (props) {
           <input className='slider__item-input js-slider-item-input'
             type='range'
             aria-label={ariaScaleName}
+            aria-valuetext={(scaleStepPrefix || scaleStepSuffix) && `${scaleStepPrefix} ${selectedValue} ${scaleStepSuffix}`}
```

Comment:
It looks like it won't read anything if there is no prefix or suffix.

Reply:
If there is no prefix or suffix the value of the input is read out, it doesn't override the value but gives a text alternative from my understanding. 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuetext

Reply:
> If there is no prefix or suffix

What about if they are both null string, as they would be in the aat?

I think it should be 
```
aria-valuetext={(scaleStepPrefix || scaleStepSuffix) ? `${scaleStepPrefix} ${selectedValue} ${scaleStepSuffix}` : null}
```

Reply:
Apologies @oliverfoster, bad grammar on my part, if there isn't a prefix AND there isn't a suffix, the aria-valueText string will be blank and the screen reader defaults to reading out the value of the input which seems to work as expected but will test your above solution.

Reply:
Whilst they both work as expected, you're absolutely right that adding null to the conditional removes the attribute from the element rather than just leaves it with an empty string, agree this is the best way forward.